orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/ExpressionAware.kt

```diff
@@ -130,7 +136,9 @@ interface ExpressionAware {
 
   private fun StageContext.augmentContext(execution: Execution): StageContext =
     if (execution.type == PIPELINE) {
-      this + mapOf("trigger" to execution.trigger, "execution" to execution)
+      this + mapOf(
+        "trigger" to mapper.convertValue(execution.trigger, object : TypeReference<Map<String, Any>>() {}),
```

Comment:
do you need this `object` bit?

Reply:
good q, i think i can use the `kotlin` `convertValue<T>` variant